gamatrix-gog.py

```diff
@@ -361,7 +367,11 @@ def parse_cmdline(argv: List[str]) -> Any:
         igdb.get_game_info(release_key)
         igdb.get_multiplayer_info(release_key)
 
-    cache.save()
+    log.info(f"DEBUG: cache state before save: {cache.data.dirty}")
+    if cache.data.dirty:
```

Comment:
If you make `Cache.save` check for `data.dirty` internally, you won't have to make this check anywhere outside the class, and if  you call it too often it won't matter.